tests/test_amclient.py

```diff
@@ -482,8 +482,8 @@ def test_get_pipelines(self):
         pipelines = objects[0]['uuid']
         resource_uri = objects[0]['resource_uri']
         assert amclient.is_uuid(pipelines)
-        assert resource_uri == ('/api/v2/pipeline/f914af05-c7d2'
-                                '-4611-b2eb-61cd3426d9d2/')
+        assert resource_uri == '/api/v2/pipeline/f914af05-c7d2' \
+                               '-4611-b2eb-61cd3426d9d2/'
```

Comment:
Oh I had never realized that you need parenthesis to do that.
I'm used to see this working:

```python
print("foo"
      "bar")
```

You can do as you prefer, this is new to me.